### PR TITLE
MAINTAINERS: Add Yan Song as a reviewer

### PR DESCRIPTION
**Pull request by changweige**
Yan Song @imeoer  is a core developer and maintainer of Nydus container image service, especially on the `nydusd` runtime/filesystem and building nydus container image via `nydus-image`, `nydusify` and [Harbor Accelleratioin Service](https://github.com/goharbor/acceleration-service) sub-project. For nydus-snapshotter he had completed features like the setting up nydus snapshots using Erofs/Fscache fs driver, which directly works on top of Linux in-kernel filesystem as container rootfs with lazyload capability, an essential feature in the upcoming nydus release. In addition, he contributed to the feature that nydus-snapshotter can directly use stargz container image to provide container rootfs with many other neat fixups.

He is eager to help community users with rich experience.

I'd like to invite him as a nydus-snapshotter reviewer if he would approve :)

Needs explicit LGTM from @imeoer and 1/3 of the nydus-snapshotter Committers:

- [x] @changweige 
- [x] @eryugey 
- [x] @imeoer

### CODE REVIEW

File: MAINTAINERS [hunk 1]
```diff
@@ -6,4 +6,5 @@
 # COMMITTERS
 # GitHub ID, Name, Email address
 changweige, Changwei Ge, changweige@gmail.com
-eryugey, Eryu Guan, eguan@linux.alibaba.com
\ No newline at end of file
+eryugey, Eryu Guan, eguan@linux.alibaba.com
+imeoer,Yan Song,imeoer@linux.alibaba.com
```
Comment (thaJeztah) [inline MAINTAINERS:10]: The description of the PR mentions "reviewer", but this change looks to add to the _committers_ section. Looks like a new section needs to be added, similar to https://github.com/containerd/project/blob/main/MAINTAINERS#L23

as you're  updating this file, perhaps you could also fix the missing newline 😅☺️

**Review by thaJeztah — CHANGES_REQUESTED** [review]
left a comment about the formatting of the file (otherwise lgtm 😅)

File: MAINTAINERS (hunk 1 above)
Comment (changweige) [reply MAINTAINERS:10]: Thanks for pointing it out. Yes, I should add a new section.

**Review by imeoer — APPROVED** [review]

File: MAINTAINERS [hunk 2]
```diff
@@ -6,4 +6,8 @@
 # COMMITTERS
 # GitHub ID, Name, Email address
 changweige, Changwei Ge, changweige@gmail.com
-eryugey, Eryu Guan, eguan@linux.alibaba.com
\ No newline at end of file
+eryugey, Eryu Guan, eguan@linux.alibaba.com
+#
+# REVIEWERS
+# GitHub ID, Name, Email address
+imeoer,Yan Song,imeoer@linux.alibaba.com
```
Comment (fuweid) [inline MAINTAINERS:13]: nit: need space after , 🥶

File: MAINTAINERS (hunk 1 above)
Comment (changweige) [reply MAINTAINERS]: fixed

File: MAINTAINERS (hunk 2 above)
Comment (changweige) [reply MAINTAINERS]: fixed

**Review by eryugey — APPROVED** [review]

**Comment by eryugey** [comment]
Or we could add Yan Song as committer as well？

**Review by fuweid — APPROVED** [review]

**Review by thaJeztah — APPROVED** [review]
LGTM

**Comment by changweige** [comment]
> Or we could add Yan Song as a committer as well？

I personally feel OK about adding Yan Some as a committer based on his remarkable contribution.  But I am not sure if it is normal to directly add a committer according to Containerd's governance.
As we have already got two votes from @containerd/reviewers, I suggest adding Yan Song as a reviewer at this stage and discussing it later on.

**Comment by changweige** [comment]
Seven days have passed. Let's merge this and welcome @imeoer to join the nydus-snapshotter maintainers team :)

**Pull request merged**